luzer/luzer.c

```diff
@@ -353,7 +356,7 @@ luaL_fuzz(lua_State *L)
 	if (!argv)
 		luaL_error(L, "not enough memory");
 
-	argv[0] = "<test name>";
+	argv[0] = argv_0;
```

Comment:
I don't see the clean up of this string. It should be freed after the end of the run.

Reply:
Thanks!

```diff
--- a/luzer/luzer.c
+++ b/luzer/luzer.c
@@ -336,7 +336,7 @@ NO_SANITIZE static void
 free_argv(int argc, char **argv)
 {
        /* Free allocated argv strings and the buffer. */
-       for (int i = 1; i < argc; i++)
+       for (int i = 0; i < argc; i++)
                free(argv[i]);
        free(argv);
 }
```